use-timescale/integrations/power-bi.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Integrate Power BI with Timescale Cloud
+excerpt: Integrate Timescale Cloud with Power BI for advanced data visualization.
+products: [cloud, mst, self_hosted]
+keywords: [Power BI, visualizations, analysis, real-time]
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+
+# Integrate Power BI with $CLOUD_LONG
+
+[Power BI][power-bi] is a business analytics tool for visualizing data, creating interactive reports, and sharing insights across an organization.
+
+This page explains how to integrate Power BI with $CLOUD_LONG using the PostgreSQL ODBC driver, so that you can build interactive reports based on the data in your $SERVICE_LONG. 
+
+## Prerequisites
+
+<IntegrationPrereqs />
+
+- Download [Power BI Desktop][power-bi-install] on your Microsoft Windows machine.
+- Install the [PostgreSQL ODBC driver][postgresql-odbc-driver].
+
+## Add your $SERVICE_LONG as an ODBC data source
+
+You use the PostgreSQL ODBC Driver to connect Power BI to $CLOUD_LONG.
+
+<Procedure>
+
+1. **Open the ODBC data sources**
+
+   1. On your Windows machine, click `Start`, then click `Control Panel`
+
+   1. In `Control Panel`, double-click `Administrative Tools`, then double-click Data Sources (ODBC).
+
+1. **Connect to your $SERVICE_LONG**
+
+   1. Under `User DSN`, click `Add`.
+   1. Choose `PostgreSQL Unicode` and click `Finish`.
+   1. Use your [connection details][connection-info] to configure the data source.
+   1. Click `Test` to ensure the connection works, then click `Save`.
+
+</Procedure>
+
+## Import the data from your your $SERVICE_LONG into Power BI  
+
+Establish a connection and import data from your $SERVICE_LONG into Power BI:
```

Comment:
It's either "You use/do... or Use/do..." in intros after all similar headers. It's different for this header and the previous one, would want this to be consistent.